src/functions-and-arrays.js

```diff
@@ -52,7 +199,23 @@ const wordsUnique = [
   'bring'
 ];
 
-function uniquifyArray() {}
+function uniquifyArray(array) {
+  if (array.length === 0){
+    return null;
+  }
+  
+  let newArray = [];
+  
+  for (i=0; i<array.length; i++){
+    if (array.indexOf(array[i])=== -1){
```

Comment:
You want to check if the word already exists in the newArray, not in the array that you're receiving
```suggestion
if (newArray.indexOf(array[i])=== -1){
```